src/options/options.js

```diff
@@ -134,18 +172,34 @@ const addListeners = async () => {
 //==========================================
 // Send test to script
 //==========================================
-const sendTestToScript = async (hasUnreadMessages) => {
-  await browser.runtime.sendNativeMessage(
-    "scriptableNotifications",
-    hasUnreadMessages
-  );
+const sendTestToScript = async (event) => {
+  switch (event) {
+    case "start":
+// TODO: event == "start"
```

Comment:
We will need those tests!

Reply:
Indeed. I just wanted to wait, that the extended payload gets consolidated.

Reply:
Done.